client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -390,6 +392,7 @@ export class CheckoutThankYou extends React.Component {
 		let failedPurchases = [];
 		let wasJetpackPlanPurchased = false;
 		let wasEcommercePlanPurchased = false;
+		let wasMarketplaceProduct = true;
```

Comment:
It looks suspicious to me that this is defaulted to `true`, since that means the thank-you page will use the marketplace variation before the data is loaded. Shouldn't it be `false` instead?

Also `wasMarketplaceProduct` doesn't sound right. `wasEcommercePlanPurchased` makes sense because it is checking if the eCommerce plan has been purchased beforehand. However, in this case we are checking the product type at present, so it should be `isMarketplaceProduct`.

Reply:
>  Shouldn't it be false instead?

Correct! :)

> wasEcommercePlanPurchased makes sense because it is checking if the eCommerce plan has been purchased beforehand.
 
Actually all thank you pages occurs post purchase. So as per my understanding this signifies that the ecommerce plan `was just purchased`. This is the same with the marketplace product, the marketplace product was just purchased.